docs/4.1/admin-guide.md

```diff
@@ -1100,12 +1094,10 @@ The recorded sessions are stored as raw bytes in the `sessions` directory under
 `log` . Each session consists of two files, both are named after the session ID:
 
 1. `.bytes` file represents the raw session bytes and is somewhat
-
     human-readable, although you are better off using [`tsh
     play`](../cli-docs/#tsh-play) or the Web UI to replay it.
 
 2. `.log` file contains the copies of the event log entries that are related to
```

Comment:
I think these are actually named differently:

```bsh
root@clear-main-auth-0:/var/lib/teleport/log/387ed398-061b-4862-9f9e-5537be748c2a/sessions/default# ls -l
total 124
-rw-r----- 1 root root 17820 Oct 23 16:35 23135814-e85f-4847-b0a2-e326dce6530b-0.chunks.gz
-rw-r----- 1 root root 30396 Oct 23 16:35 23135814-e85f-4847-b0a2-e326dce6530b-0.events.gz
-rw-r----- 1 root root   204 Oct 23 16:35 23135814-e85f-4847-b0a2-e326dce6530b.index
-rw-r----- 1 root root  5768 Oct 23 16:30 88e597cf-63cb-405b-a9e7-1f93671b62dc-0.chunks.gz
-rw-r----- 1 root root 18502 Oct 23 16:30 88e597cf-63cb-405b-a9e7-1f93671b62dc-0.events.gz
-rw-r----- 1 root root   204 Oct 23 16:30 88e597cf-63cb-405b-a9e7-1f93671b62dc.index
-rw-r----- 1 root root  8252 Oct 23 16:19 cdb5ec06-f5b0-11e9-a722-128806646f6b-0.chunks.gz
-rw-r----- 1 root root 16750 Oct 23 16:19 cdb5ec06-f5b0-11e9-a722-128806646f6b-0.events.gz
-rw-r----- 1 root root   204 Oct 23 16:19 cdb5ec06-f5b0-11e9-a722-128806646f6b.index
```

so `.bytes` files are actually `.chunks.gz` and `.log` is actually `.events.gz`

Reply:
Are those just just the compressed versions?